src/components/sellLuggageAllowance/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import {withRouter} from 'react-router-dom';
+import './style.scss'
+//components
+import SellInfo from '../sellInfo'
+
+class SellLuggageAllowance extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
We should save all the states in the sellInfo component as they are not used in this component. In fact they can just be local variables in that component